### PR TITLE
Add Hackolade

### PR DESCRIPTION
**Pull request by hackolade**
Add application to Closed source

### CODE REVIEW

**Comment by sindresorhus** [comment]
https://github.com/sindresorhus/awesome-electron/blob/master/contributing.md

> If you're submitting a closed source app, include evidence of it being built with Electron.

> Additions should be added to the bottom of the relevant section.

File: readme.md [hunk 1]
```diff
@@ -166,6 +166,7 @@ Made with Electron.
 - [Pubu](https://pubu.im) - Real-time chat for team communication. *(Chinese)*
 - [BearyChat](https://bearychat.com) - Team messaging service. *(Chinese)*
 - [MongoDB Compass](https://www.mongodb.com/products/compass) - Official MongoDB app.
+- [Hackolade](https://hackolade.com) - Visual modeling of MongoDB schemas, and other NoSQL document data stores.
```
Comment (sindresorhus) [inline readme.md]: The website doesn't load for me.

Comment (sindresorhus) [inline readme.md]: Needs a shorter description.

**Comment by hackolade** [comment]
Oops.  Fixed both issues reported.  Sorry about that.

**Comment by hackolade** [comment]
Regarding evidence, here's a screenshot of the about box of the application at http://http://hackolade.com/assets/img/AboutElectron.png

**Comment by sindresorhus** [comment]
Alright. You need to fix the merge conflict (That does not mean opening a new pull request).

**Comment by hackolade** [comment]
It seems that I don't have the proper rights to do that.  Sorry.

**Comment by sindresorhus** [comment]
https://www.google.no/search?q=stackoverflow+fix+merge+conflict+github&oq=stackoverflow+fix+merge+conflict+github&aqs=chrome..69i57j69i64.7310j0j4&sourceid=chrome&ie=UTF-8

**Comment by pdesmarets** [comment]
It is true that I'm not very proficient in Git.  However, I thought that I had done my homework and reviewed many of these links before posting my previous message.  But I keep getting messages, both on github.com and in GitHub Desktop that 'Only those with write access to this repository can merge pull requests.' and 'It seems that you do not have permission to push your changes to this repository.'  

Sorry for my ignorance, and thank you in advance for your patience.

**Comment by sindresorhus** [comment]
I'm asking you to "fix the merge conflict", not "merge the pull request".

**Comment by spiliopoulos** [comment]
@hackolade in order to fix the merge conflict you should fetch the latest changes from upstream (this repo) to your fork (hackolade/awesome-electron). Then you should merge your fork's upstream master to your branch for which you requested a pull (patch-1). That will result in a merge conflict. You should edit the files with the conflict, fix them, commit them and push.

Something like this:

    $git fetch upstream
    $git checkout patch-1
    $git merge upstream/master
    File Foo has conflicts
    $vim Foo #Fix conflicts and save
    $git add Foo
    $git commit #Merge of branch upstream/master into patch-1
    $git push

The last push will update the pull request with your fixes for the merge conflicts, making it possible for sindresorhus to pull it.

**Comment by hackolade** [comment]
Thanks @spiliopoulos I appreciate it.  I know just enough to be dangerous here, and I don't want to do anything stupid.  I'll ask one of the developers to make this change for me.  It seemed simple enough, I didn't realize I was so handicapped ;-) Cheers!

**Comment by spiliopoulos** [comment]
No worries. You seemed lost, I thought I could help. We all have been there and we will all be there again.

**Pull request closed without merging**